hack/test-go.sh

```diff
@@ -191,8 +191,8 @@ if [[ -n "${junit_report}" ]]; then
     set -o pipefail
 
     if [[ -s "${test_error_file}" ]]; then
-        os::log::warning "\`go test\` had the following output to stderr:"
-        cat "${test_error_file}"
+        os::log::warning "\`go test\` had the following output to stderr:
+$( cat "${test_error_file}") "
```

Comment:
indentation?

Reply:
> indentation?

it's in quotes.  I don't think he has a choice.

Reply:
I will atone for my indentation sins, but David is correc